File: problem_2.py
fib = [1, 2]
i = 2
n = 0

while n < 4000000:
	n = fib[i - 1] + fib[i - 2]
	fib.append(n)
	i += 1

print sum(filter(lambda x: x % 2 == 0, fib))

File: problem_2.c
#include <stdio.h>

int main(void) {
    int n = 0;
    int fib1 = 1;
    int fib2 = 2;
    int sum = fib2;

    while (n < 4000000) {
        n = fib1 + fib2;
        fib1 = fib2;
        fib2 = n;
        if (n % 2 == 0) {
            sum += n;
        }
    }
    printf("%d\n", sum);
}
